test/focus-trap.test.js

```diff
@@ -65,12 +65,9 @@ describe('FocusTrap', function() {
 
   it('returns focus when it is lost', function(done) {
     let component = DOM.render(<FocusTrap />, document.body)
-    let el = DOM.findDOMNode(component.focus)
+    let el = DOM.findDOMNode(component).querySelector('.focus-trap')
```

Comment:
super minor thought: what about passing in `<FocusTrap className="focus-trap" />` above, so that it's not tied directly to the default prop?

Reply:
Great suggestion, 👍